### PR TITLE
correct the return information in scheduler.go

### PR DESCRIPTION
**Pull request by mysunshine92**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

 /kind bug

**What this PR does / why we need it**:
correct the return information in /pkg/scheduler/scheduler.go

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by mysunshine92** [comment]
cc @sig-scheduling-maintainers

File: pkg/scheduler/scheduler.go [hunk 1]
```diff
@@ -287,7 +287,7 @@ func (sched *Scheduler) schedule(pod *v1.Pod) (core.ScheduleResult, error) {
 		sched.recordSchedulingFailure(pod, err, v1.PodReasonUnschedulable, err.Error())
 		return core.ScheduleResult{}, err
 	}
-	return result, err
+	return result, nil
```
Comment (ahg-g) [inline pkg/scheduler/scheduler.go]: that is not ab bug, err is going to be nil here anyways, right?

Comment (draveness) [reply pkg/scheduler/scheduler.go]: /remove-kind bug
/kind cleanup
/lgtm

**Pull request merged**